applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -82,9 +82,22 @@ static std::ostream& pythonToSofaDataString(PyObject* value, std::ostream& out)
         return out << PyString_AsString(value) ;
     }
 
+    /// Unicode are converted to string.
+    if(PyUnicode_Check(value))
+    {
+        PyObject* tmpstr = PyUnicode_AsUTF8String(value);
+        out << PyString_AsString(tmpstr) ;
+        Py_DECREF(tmpstr);
+
+        return out;
+    }
 
     if( PySequence_Check(value) )
     {
+        if(!PyList_Check(value))
```

Comment:
@damienmarchal given the numerous side-effect we keep on discovering, I'd suggest leaving the warning until things settle down a little bit.